stock_management/tmp_win_utils.py

```diff
@@ -0,0 +1,36 @@
+from stock_management.layouts import get_test_layout
+import PySimpleGUI as sg
+import time
+
+
+def test_session(
+    test_events=[],
+    test_args=[],
+    timeout=None,
+):
+    layout = get_test_layout()
+    window = sg.Window("Test", layout)
+    window.finalize()
+
+    tstat = time.time()
+    while True:
+        event, values = window.read(timeout=100)
+        if event == sg.WIN_CLOSED:
+            break
+        elif event == "-but_exit_test-":
+            print("exit win Test")
+            break
+        elif event == "-but_confirm_test-":
+            # values contains the values in the different widgets of the windows. It is not the returned value of a read event.
```

Comment:
Notice the usage of `values`. The variable `values`, does not contain the return state of the function `window.read`, but rather all the values contained into the widget of the window that you created. In this case, it contains the text in the Inpute window.